subs/build_miner.sh

```diff
@@ -65,7 +65,7 @@ init_update_external_modules
 
 autoreconf -i
 
-./configure --prefix=$BUILD_DIR/output CFLAGS="-I$BUILD_DIR/output/include" LDFLAGS="-L$BUILD_DIR/output/lib" CPPFLAGS="-I$BUILD_DIR/output/include" LIBCURL_CFLAGS="-I$BUILD_DIR/output/include/curl" LIBCURL_LIBS="-L$BUILD_DIR/output/lib -lcurl" LIBUSB_CFLAGS="-I$BUILD_DIR/output/include/libusb-1.0" LIBUSB_LIBS="-L$BUILD_DIR/output/lib -lusb-1.0" --enable-scrypt --enable-keccak --enable-bflsc --enable-bitforce --enable-icarus --enable-modminer --enable-ztex --enable-avalon --disable-adl --enable-opencl --enable-bitfury --enable-hashfast --enable-klondike --enable-cointerra --enable-blockerupter --enable-hashratio --enable-avalon2 --enable-drillbit
+./configure --prefix=$BUILD_DIR/output CFLAGS="-I$BUILD_DIR/output/include" LDFLAGS="-L$BUILD_DIR/output/lib" CPPFLAGS="-I$BUILD_DIR/output/include" LIBCURL_CFLAGS="-I$BUILD_DIR/output/include/curl" LIBCURL_LIBS="-L$BUILD_DIR/output/lib -lcurl" LIBUSB_CFLAGS="-I$BUILD_DIR/output/include/libusb-1.0" LIBUSB_LIBS="-L$BUILD_DIR/output/lib -lusb-1.0" PKG_CONFIG_PATH=$BUILD_DIR/output/lib/pkgconfig --enable-scrypt --enable-keccak --enable-bflsc --enable-bitforce --enable-icarus --enable-modminer --enable-ztex --enable-avalon --disable-adl --enable-opencl --enable-bitfury --enable-hashfast --enable-klondike --enable-cointerra --enable-blockerupter --enable-hashratio --enable-avalon2 --enable-drillbit
```

Comment:
Please quote the path in the new arg or it will fail if built in a path that contains spaces, e.g.:

```
PKG_CONFIG_PATH="$BUILD_DIR/output/lib/pkgconfig"
```

Reply:
This change as-is works for BFGMiner but not CGMiner - CGMiner still needs the following change:

```
CFLAGS="-I$BUILD_DIR/output/include -I$BUILD_DIR/output/include/libusb-1.0"
```

Reply:
Whoops, thanks, I've changed that now!